packages/components/src/dropdown-button/dropdown-button.tsx

```diff
@@ -53,6 +53,10 @@ const iconStyles = cva({
         'text-neutral-100 dark:text-white-100',
         '[&>path[fill="#E7EAEE"]]:dark:fill-neutral-80',
       ],
+      darkGrey: [
+        'text-neutral-100 dark:text-white-100',
+        '[&>path[fill="#E7EAEE"]]:dark:fill-neutral-80',
+      ],
```

Comment:
What's the difference of this variant to the grey one?

Reply:
this styles affect icon, button styling are in the `<Button />`

Reply:
Nevermind. I'm just tired. Makes total sense. I thought i was looking at `<Button />` component. 🤦